api/scripts/wazuh-apid.py

```diff
@@ -213,7 +213,6 @@ def start(params: dict):
 
     # Add error handlers to format exceptions
     app.add_error_handler(ExpectFailedException, error_handler.expect_failed_error_handler)
-    app.add_error_handler(jwt.exceptions.PyJWTError, error_handler.jwt_error_handler)
```

Comment:
The exception `jwt.exceptions.PyJWTError` is never thrown because we are [catching it](https://github.com/wazuh/wazuh/blob/b2bea025d4f70360b66be77fddc743cb4eb80064/api/api/authentication.py#L299C1-L300C51) and raising an `Unauthorized` error.

The other place where `jwt_decode` is called is when logging information about the request but the exception is dismissed if it occurs.

Hence this error handler is not used and can be safely removed.

Reply:
Great!